msteams-platform/messaging-extensions/api-based-microsoft-entra.md

```diff
@@ -0,0 +1,342 @@
+
+    1. Select **Add application**.
+
+       A message pops up on the browser stating that the authorized client app was added.
+
+       :::image type="content" source="../assets/images/authentication/teams-sso-tabs/update-app-auth-msg.png" alt-text="Client application added message":::
+
+       The authorized app's client ID displays on the page.
+
+       :::image type="content" source="../assets/images/authentication/teams-sso-tabs/client-app-added.png" alt-text="Client app added and displayed":::
+
+> [!NOTE]
+> You can authorize more than one client application. Repeat the steps of this procedure for configuring another authorized client application.
+
+You've successfully configured app scope, permissions, and client applications. Ensure that you note and save the application ID URI. Next, you configure the access token version.
+
+## Update app manifest
+
+Update the following properties in the app manifest file:
+
+* `webApplicationInfo`: The `webApplicationInfo` property is used to enable SSO for your app to help app users access your API-based message extension app seamlessly. The application ID URI that you registered in Microsoft Entra ID is configured with the scope of the API you exposed. For more information, see [webApplicationInfo](../resources/schema/manifest-schema.md#webapplicationinfo).
+
+   &nbsp;&nbsp;:::image type="content" source="../assets/images/authentication/teams-sso-tabs/sso-manifest.png" alt-text="Screenshot shows the app manifest configuration.":::
+
+* `microsoftEntraConfiguration`: Enables Single sign-on authentication for your app. Configure the `supportsSingleSignOn` property to `true` to support SSO and reduce the need for multiple authentications. If the property is set to `false` or is left empty, the user can't upload the app to Teams and the app fails validation.
+
+To configure app manifest:
+
+1. Open the API-based message extension app.
+2. Open the app manifest folder.
+
+    > [!NOTE]
+    >
+    > * The app manifest folder should be at the root of your app folder. For more information, see [Create a Microsoft Teams app package](../concepts/build-and-test/apps-package.md).
+    > * For more information on learning how to create a manifest.json, see [the app manifest schema](../resources/schema/manifest-schema.md).
+
+1. Open the `manifest.json` file
+1. Add the following code snippet to the app manifest file:
+
+   **webApplicationInfo**
+
+    ```json
+    "webApplicationInfo":
+    {
+    "id": "{Microsoft Entra AppId}",
+    "resource": "api://subdomain.example.com/{Microsoft Entra AppId}"
+    }
+    ```
+
+    where,
+    * `{Microsoft Entra AppId}` is the app ID you created when you registered your app in Microsoft Entra ID. It's the GUID.
+    * `api://subdomain.example.com/{Microsoft Entra AppId}` is the application ID URI that you registered when creating scope in Microsoft Entra ID.
+
+    **MicrosoftEntraConfiguration**
+
+    ```json
+    "authorization": {
```

Comment:
the authorization object should be added to composeExtension node. 
 
pls add this information. otherwise it will not be clear to the dev where to add the authorization node.

Reply:
Added the required information.